easybuild/framework/easyblock.py

```diff
@@ -2510,7 +2510,7 @@ def permissions_step(self):
                 perms |= stat.S_IROTH
                 self.log.debug("Also ensuring read permissions for others on install dir (no group specified)")
 
-            umask = build_option('umask')
+            umask = self.cfg['umask']
```

Comment:
@pescobar This should certainly be something like `self.cfg['umask'] or build_option('umask')` (but that won't solve your problem of it not being picked up).